arrMethodsLab.js

```diff
@@ -0,0 +1,409 @@
+// # Exercises
+//
+// 1. Given an array of numbers, write a function that returns a new array where every element is incremented by 10.
+console.log("---------- #1 ----------")
+let arr1 = [1, 2, 3, 4, 5]
+let plusTen = []
+
+arr1.forEach((currentElem) => {
+  plusTen.push(currentElem + 10)
+}
+)
+
+console.log(plusTen)
+
```

Comment:
Right idea. Do you know what array method returns a new array? How could you implent that method